ext/hivemq-edge-openapi-2024.8-proposal.yaml

```diff
@@ -0,0 +1,5561 @@
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              examples:
+                metric-sample:
+                  description: Example response with metrics listed.
+                  summary: Metric Sample
+                  value:
+                    sampleTime: 2023-06-28T11:39:12.789+01
+                    value: 0
+              schema:
+                $ref: '#/components/schemas/DataPoint'
+          description: Success
+      summary: Obtain the latest sample for the metric requested
+      tags:
+        - Metrics
+        - Metrics Endpoint
+  /api/v1/management/domain/tags:
+    get:
+      description: Get the list of all domain tags created in this Edge instance
+      operationId: get-domain-tags
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+          description: Success
+      summary: Get all domain tags
+      tags:
+        - Domain
+  /api/v1/management/domain/tags/schema:
+    get:
+      description: Get the data schema associated with the specified tags
+      operationId: get-tag-schemas
+      parameters:
+        - description: The list of tags to query the schema from
+          in: query
+          name: tags
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TagSchema'
+          description: Success
+      summary: Get data schemas
+      tags:
+        - Domain
+  /api/v1/management/domain/topics:
+    get:
+      description: Get the list of all topic tags created in this Edge instance
+      operationId: get-domain-topics
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+          description: Success
+      summary: Get all topic tags
+      tags:
+        - Domain
+  /api/v1/management/domain/topics/schema:
+    get:
+      description: Get the data schema associated with the specified topics
+      operationId: get-topic-schemas
+      parameters:
+        - description: The list of topics to query the schema from
+          in: query
+          name: topics
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TagSchema'
+          description: Success
+      summary: Get data schemas
+      tags:
+        - Domain
+
+  /api/v1/management/client/topic-samples:
```

Comment:
Why is this needed?

Reply:
why is this path under the root '/client' but topics are under the root '/domain' ?